docs/userguide/about/install.md

```diff
@@ -245,7 +245,27 @@ uv add earth2studio --extra fuxi
 ::::
 :::::
 :::::{tab-item} GraphCast
```

Comment:
Fine to just have one install group for all graphcast models, like what we do for pangus. Keeps it simple since its all the same

Reply:
Great thanks